ExilenceNextApp/public/i18n/en/common.json

```diff
@@ -65,10 +74,28 @@
     "icon_latest": "Only show the latest snapshot",
     "icon_comparison": "Difference for the two latest snapshots",
     "icon_fetched_from": "Icon fetched from pathofexile.com",
-    "income_based_on": "Based on last hour",
-    "income_based_on_tooltip": "Income is calculated based on snapshot values over the last hour",
+    "net_worth_session_started_at_tooltip": "Net worth session is started at",
+    "time_since_last_snapshot_tooltip": "Time since last snapshot",
+    "income_based_on_session_duration": "Based on session duration",
+    "income_based_on_last_pause": "Based on last pause",
+    "income_based_on_last_offline": "Based on last offline",
+    "income_based_on_last_inactive": "Based on last inactivity",
+    "income_based_on_last_hour": "Based on last hour",
+    "income_based_on_session_duration_tooltip": "Income is calculated based on snapshot values over the session duration",
+    "income_based_on_last_pause_tooltip": "Income is calculated based on snapshot values over the end of the last pause",
+    "income_based_on_last_offline_tooltip": "Income is calculated based on snapshot values over the end of the last offline",
+    "income_based_on_last_inactive_tooltip": "Income is calculated based on snapshot values over the end of the last inactivity",
+    "income_based_on_last_hour_tooltip": "Income is calculated based on snapshot values over the last hour",
+    "income_based_on_session_duration_tooltip_short": "Session duration",
+    "income_based_on_last_pause_tooltip_short": "Since last pause",
+    "income_based_on_last_offline_tooltip_short": "Since last offline",
+    "income_based_on_last_inactive_tooltip_short": "Since last inactivity",
+    "income_based_on_last_hour_tooltip_short": "Since last hour",
+    "critial_stashtyp_unique_tooltip": "Attention, a separate request is made for each item in the unique stashtab. This can significantly increase snapshot duration.",
+    "critial_stashtyp_map_tooltip": "Attention, spearate requests are made for all tiles in the map stash tab, which are not atlas maps. This can significantly increase snapshot duration.",
```

Comment:
```suggestion
    "critical_stashtype_map_tooltip": "Separate requests are made for every non standard map in the map stash tab. This can significantly increase snapshot duration.",
```